docs/LwM2M_Client/Nordic/nRF7002DK.md

```diff
@@ -40,14 +40,14 @@ Enter the command line interface on your machine and change the directory to the
 
 #### Part 3: Compiling the example
 
-0. Connect the nRF7002 DK board to a USB port of your machine.
+0. Connect the {{ nRF7002DK_board_name }} board to a USB port of your machine.
 0. Set West manifest path to `Anjay-zephyr-client/demo`, manifest file to `west-nrf.yml`, and do `west update`:
     ```
     west config manifest.path Anjay-zephyr-client/demo
     west config manifest.file west-nrf.yml
     west update
     ```
-0. Compile and flash the project for **nRF7002 DK**:
+0. Compile and flash the project for **{{ nRF7002DK_board_name }}**:
     ```
     cd Anjay-zephyr-client/demo
     west build -b nrf7002dk_nrf5340_cpuapp
```

Comment:
If we want to unify those files, what about flashing commands?

Reply:
This is more complex since yaml file do not understand multiline variables :/
Unifying whole file and making this automatized by script is a lot of work and is planned by @Mierunski in another task